ibex_configs.hjson

```diff
@@ -0,0 +1,18 @@
+{
+  small-fastmult : {
+    RV32E                    : 0
+    RV32M                    : 1
+    BranchTargetALU          : 0
+    WritebackStage           : 0
+    MultiplierImplementation : "fast"
+  }
+
+  maxperf-scmult : {
```

Comment:
The naming of the multipliers is confusing here. Is `fast` slower than `sc`? I know this matches our code currently, but maybe we don't need to deepen the use of these confusing terms. Can we maybe use the # of cycles as naming? E.g. `4cmul`, or `1cmul`?